src/system/utils.ts

```diff
@@ -14,47 +14,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- // notice_end
+// notice_end
 
-export function removeAll(arr, item) {
-    for(let i = arr.length; i--;) {
-        if(arr[i] === item) {
+export function removeAll(arr: any[], item: any) {
+    for (let i = arr.length; i--;) {
+        if (arr[i] === item) {
             arr.splice(i, 1);
         }
     }
 }
 
-export function isString(value) {
+export function isString(value: any): boolean {
```

Comment:
this function is only needed if we're using it to check parameters outside of this library.  If it's only used to check internal values, TS means we don't need it